feedback/lib/src/l18n/translation.dart

```diff
@@ -223,7 +223,9 @@ class GlobalFeedbackLocalizationsDelegate
   static const LocalizationsDelegate<FeedbackLocalizations> delegate =
       GlobalFeedbackLocalizationsDelegate();
 
-  static final _supportedLocales = <Locale, FeedbackLocalizations>{
+  /// Returns a dict of all supported locales.
+  /// Override this member to provide your own localized strings.
+  static final supportedLocales = <Locale, FeedbackLocalizations>{
```

Comment:
You can't override `static` properties on a class.
```suggestion
  final supportedLocales = <Locale, FeedbackLocalizations>{
```